lighthouse-core/test/results/sample_v2.json

```diff
@@ -895,31 +890,25 @@
         "items": [
           {
             "url": "http://localhost:10200/dobetterweb/dbw_tester.html",
-            "total": 1044.3820000000003,
-            "scripting": 963.7010000000001,
-            "scriptParseCompile": 2.777
+            "total": 1450.7370000000005,
+            "scripting": 1252.0330000000006,
+            "scriptParseCompile": 6.970000000000001
           },
           {
             "url": "Unattributable",
-            "total": 279.87100000000004,
-            "scripting": 9.61,
-            "scriptParseCompile": 0.092
-          },
-          {
-            "url": "http://localhost:10200/zone.js",
-            "total": 107.086,
-            "scripting": 90.81299999999999,
-            "scriptParseCompile": 1.674
+            "total": 142.05800000000022,
+            "scripting": 15.010999999999997,
+            "scriptParseCompile": 0.5860000000000001
           },
           {
-            "url": "http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js",
```

Comment:
did we remove jquery here?

Reply:
It's still on the page. The timing is so small here, perhaps it's just on the threshold of being included consistently.